broker-daemon/models/block-order.js

```diff
@@ -129,6 +129,38 @@ class BlockOrder {
     return counterCommonAmount.times(this.counterCurrencyConfig.quantumsPerCommon).round(0).toString()
   }
 
+  /**
+  * Convenience getter for outboundAmount
+  * @return {String} String representation of the amount of currency we will send outbound for the order
+  */
+  get outboundAmount () {
+    return this.isBid ? this.counterAmount : this.baseAmount
+  }
+
+  /**
+  * Convenience getter for inboundAmount
+  * @return {String} String representation of the amount of currency we will receive inbound for the order
+  */
+  get inboundAmount () {
+    return this.isBid ? this.baseAmount : this.counterAmount
+  }
+
+  /**
+  * Get the symbol of the currency we will receive inbound if the order is completed
```

Comment:
@martineehrlich we should remove `if the order is completed` to the docs for these functions

Reply:
Remove the text in `order.js` too